isdown/assets/dashboards/isdown_overview.json

```diff
@@ -0,0 +1,64 @@
+{
+  "title": "IsDown Dashboard",
+  "description": "",
```

Comment:
please include a description on the value of the dashboard

Reply:
Thank you, @emarsha94. Added a description.